app/src/main/java/org/hisp/dhis/android/dataentry/reports/ReportsAdapter.java

```diff
@@ -58,15 +63,21 @@ void swapData(@NonNull List<ReportViewModel> reports) {
         reportViewModels.clear();
         reportViewModels.addAll(reports);
 
-        // ToDo: improve performance of RecyclerView
         notifyDataSetChanged();
     }
 
+    interface OnReportViewModelClickListener {
+        void onClick(@NonNull ReportViewModel reportViewModel);
+    }
+
     static class ReportViewHolder extends RecyclerView.ViewHolder {
 
 //        @BindView(R.id.container_status_icon)
```

Comment:
It depends if decide to keep status icon in the list. For now it is there just as a reminder that we can implement it.